### ISSUE TITLE
Scanning speed is amazingly slow

### ISSUE THREAD

**Issue by PikaPie** [favll/pogom#342]
I'm currently scanning 979 locations with 50 accounts. However the scanning speed is amazingly slow. I need for the entire area about 25 minutes. I initially had only 30 accounts running with approximately the same scanning duration. Only about 1 or 2 accounts frequently generate errors, the rest is working fine.  This is baffling for me, since the math predicts that with 50 accounts I should only need about 3 - 4 minutes for the entire scan. Any idea what might be the problem?

**Comment by qwerty0123** [comment]
Can you show me a bit of your log? Like 200 lines or so

**Comment by PikaPie** [comment]
The log is really not much of interest. Especially since I removed the faulty accounts today, so there are no error messages at all anymore. You basically see in the first lines how pogom logs into the accounts and then scans all the determined locations. Where each subsequent scans is delayed by about 0.2 - 4 seconds. A redditor suggested that the issue might be related to the process of writing the collected information into the database and said that his problem was solved by switching to MySQL and modifying certain parts of the code accordingly.
